component/api/src/main/java/org/exoplatform/portal/pom/data/PortalData.java

```diff
@@ -104,7 +118,11 @@ public PortalData(String storageId, // NOSONAR
          skin,
          portalLayout,
          false,
-         redirects);
+         redirects,
+         displayed,
+         displayOrder,
+         bannerUploadId,
```

Comment:
The `bannerUploadId` shouldn't be cached in Storage layer, isn't it ? BTW, shouldn't we delete `PortalData` class. In fact, it was used before as an object to put data in a cache with an adequate serialization manner (for clustering). I think we can rely on PortalConfig on this to simplify maintainability of layers. WDYT ?

Reply:
PortalData is used in many locations, IMO removing it and refactoring layers is not the scope of this MIP, this needs a study I think. cc @rdenarie

Reply:
ok, no problem, it was just a suggestion in case you want to do some refactoring to clean the API.
What about the `bannerUploadId`, is it ok for you to delete it from `PortalData` knowing that it won't be used/cached ?

Reply:
deleted